ext/ruby.c

```diff
@@ -1,5 +1,6 @@
 #include "ruby/ruby.h"
 #include "stdint.h"
+#include "intern.c"
```

Comment:
I don’t think we need this. The makefile seems to be able to find all c files just fine.

Reply:
Doesn't build without it for me.

Reply:
Wait... if I add `#include "ruby/ruby.h"` to `intern.c` then it does... strange.

Reply:
I feel like something strange is going on here.

Reply:
Oh, I see, `ruby/intern.h` doesn't include `ruby/ruby.h` so I have to include for it.